aws_marketplace/using_algorithms/autogluon/autogluon_tabular_marketplace.ipynb

```diff
@@ -123,8 +123,10 @@
    "source": [
```

Comment:
<p>Please use boto3 rather than wget. We also have an AWS-hosted copy of the original Adult dataset. Can you use the following and then split into train and test sets?</p><pre class="ql-syntax" spellcheck="false">import boto3
s3 = boto3.client("s3")
s3.download_file("sagemaker-sample-files", "datasets/tabular/uci_adult/adult.data", "adult.data")
</pre><p>(This <code>.data</code> file is a csv file)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Thank you for the comment. Changed the method to boto3 but remained the source as is because the sagemaker version doesn't have header.

Reply:
<p>Can you elaborate on what would be useful to have in a header? It may help other users as well.</p>